package.json

```diff
@@ -20,7 +20,6 @@
     "emailjs-com": "^3.2.0",
     "express": "^4.19.2",
     "file-saver": "^2.0.5",
-    "framer-motion": "^11.3.17",
```

Comment:
why you remove these ...also check some are in below

Reply:
Updated.